common/templates/options.php

```diff
@@ -79,6 +79,27 @@
                     <?php endif; ?>
                 </td>
             </tr>
+            <tr valign="top">
+                <th scope="row">
+                    <label><?php _e('Order status for invitations', 'webwinkelkeur'); ?></label>
+                </th>
+                <td>
+                    <fieldset>
+                        <div style="height: 150px; overflow: auto;">
+                            <?php foreach (wc_get_order_statuses() as $key => $label): ?>
+                                <div>
```

Comment:
Instead of `div`, use `label`, so that clicking the text also triggers the checkbox.